src/v/cluster/controller_backend.cc

```diff
@@ -1352,7 +1352,19 @@ ss::future<std::error_code> controller_backend::update_partition_replica_set(
       replicas,
       rev,
       [this, rev, &replicas, &ntp](ss::lw_shared_ptr<partition> p) {
```

Comment:
This is just removing duplicated code. There were two completely separate methods to perform configuration replacement. Since now we track the revision of added nodes outside of raft and we already use method assigning revisions we can simply remove the old method to keep things simpler and consistent.